### PR TITLE
Linux fixes

### PR DESCRIPTION
**Pull request by CamSoper**

### CODE REVIEW

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Configuring-NuGet-Behavior.md](https://github.com/CamSoper/docs.microsoft.com-nuget/blob/helping-out/docs/consume-packages/Configuring-NuGet-Behavior.md) | :white_check_mark:Succeeded | |
[docs/includes/install-cli.md](https://github.com/CamSoper/docs.microsoft.com-nuget/blob/helping-out/docs/includes/install-cli.md) | :white_check_mark:Succeeded | |
[docs/install-nuget-client-tools.md](https://github.com/CamSoper/docs.microsoft.com-nuget/blob/helping-out/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C30%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201803302342427236-810%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=CJbAX%2FEXZgNpOfJED4RagUoBhuXjZVAyVjZz4LsscOs%3D&st=2018-03-30T23%3A38%3A42Z&se=2018-04-30T23%3A43%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by CamSoper** [comment]
@kraigb - Does the CI build stage these PRs for review somewhere?  Like in the other repos?

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Configuring-NuGet-Behavior.md](https://github.com/CamSoper/docs.microsoft.com-nuget/blob/helping-out/docs/consume-packages/Configuring-NuGet-Behavior.md) | :white_check_mark:Succeeded | |
[docs/includes/install-cli.md](https://github.com/CamSoper/docs.microsoft.com-nuget/blob/helping-out/docs/includes/install-cli.md) | :white_check_mark:Succeeded | |
[docs/install-nuget-client-tools.md](https://github.com/CamSoper/docs.microsoft.com-nuget/blob/helping-out/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C31%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201803310003246932-810%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dlqcbGiciIpF9tIaq96bWV18%2Fm7hsOb1waFfXw5sXaE%3D&st=2018-03-30T23%3A59%3A20Z&se=2018-05-01T00%3A04%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/install-nuget-client-tools.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Installation:
 [!INCLUDE[install-cli](includes/install-cli.md)]
 
 > [!Tip]
-> Use `nuget update -self` to update an existing nuget.exe to the latest version.
+> Use `nuget update -self` to update an existing nuget.exe to the latest version on Windows.
```
Comment (kraigb) [inline docs/install-nuget-client-tools.md]: I'd say, "On Windows, use `nuget update -self`...to someone doesn't read the whole sentence before figuring that out.

**Comment by kraigb** [comment]
@CamSoper You can use review.docs.microsoft.com/nuget/...?branch=helping-out.

**Comment by karann-msft** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/consume-packages/Configuring-NuGet-Behavior.md](https://github.com/CamSoper/docs.microsoft.com-nuget/blob/helping-out/docs/consume-packages/Configuring-NuGet-Behavior.md) | :white_check_mark:Succeeded | |
[docs/includes/install-cli.md](https://github.com/CamSoper/docs.microsoft.com-nuget/blob/helping-out/docs/includes/install-cli.md) | :white_check_mark:Succeeded | |
[docs/install-nuget-client-tools.md](https://github.com/CamSoper/docs.microsoft.com-nuget/blob/helping-out/docs/install-nuget-client-tools.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C4%5C2%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201804021646486741-810%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cYlF9Zvyo9FQGv0qOHT8xeoCQiLPgmik47aAur7xzkE%3D&st=2018-04-02T16%3A42%3A42Z&se=2018-05-03T16%3A47%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**